### ISSUE TITLE
[A11Y] All Editorial View - Settings area, the MCE rich text area is not readable with a screenreader and cannot access some format options

### ISSUE THREAD

**Issue by Tribunal33** [pkp/pkp-lib#12721]
## Issue Description
Tiny MCE rich text fields are not readable by screenreader.  Some auditors have mentioned this as a big problem.  Such as
> "The most troubling aspect about Narrator and this site is the inability of Narrator to read the rich text areas which comprise many of the editing in this site.” 

This lack of announcing the label for TinyMCE was mentioned breifly in #12707 but it was specific to a single area.  This would be to fix how MCE creates these rich text fields and how to improve them.  

<img width="371" height="80" alt="Image" src="https://github.com/user-attachments/assets/fbfdf376-051b-49f6-a492-8056148b2bf5" />

Additionally it was noted that you cannot access the format options within this field.  I am not sure how they were able to access even some of them I couldn't get the focus onto the tinyMCE format options.  

As noted from this auditor:
>  “If the arrow keys are used in the format tool bar, only the bold italics and underline are repeated by NVDA."

<img width="263" height="58" alt="Image" src="https://github.com/user-attachments/assets/da75c6f5-08c0-4c11-816a-aa0016a1360a" />

## How to reproduce / Current behavior
Navigate to any Setting area.  In particular we will focus on the Workflow Setting > Submission > Author Guidance section.

## What application are you using?
 OJS, OMP or OPS version 3.5.0-3

## Stack used 
- Device: Laptop
- OS: Windows 11
- Browser: Chrome
- Version: Latest

## Testing method
Manual using screenreader NVDA

## Action
- [ ] Add the proper configuration settings to tinyMCE for making the rich-text-area screenreader friendly.  Not sure if the configuration fix will allow the formating options also be fixed but if not
- [ ] add ability to tab through the format options with keyboard

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.

here was a code snippet that suggested how it could be improved

`tinymce.init({
  selector: 'textarea',
  // Sets aria-label on the body inside the iframe or title on the iframe
  iframe_aria_text: 'Rich Text Area - Comments',
  // ... other settings
});
`
